doc/nrf/protocols/matter/end_product/security.rst

```diff
@@ -161,6 +161,79 @@ This is a reference configuration that can be modified in the production firmwar
        To use the Oberon backend for specific cryptographic operations supported by both drivers, disable those operations in the CRACEN driver, as it takes priority when both are enabled.
        See the :ref:`nrf_security_drivers` documentation for more information.
 
+.. _matter_platforms_security_kmu:
+
+nRF54L Key Management Unit (KMU)
+================================
+
+nRF54L series devices include :ref:`ug_nrf54l_crypto_kmu_cracen_peripherals` that can be used to store cryptographic keys in Matter.
+In this solution, the keys are stored within the available slots in the :ref:`ug_nrf54l_crypto_kmu_slots` range that are not reserved for current and future |NCS| use cases.
```

Comment:
After looking at the doc preview, i think what we are missing here is the description that the KMU slots are used to store Matter Fabric credentials when given X KConfig is enabled - and let's document for now, that this is not enabled by default. Thoughts?

Reply:
Right, this description is missing, will add it. Regarding Kconfig we have a line there; `to use this feature set...`, but it seems to be ok to add a note that this is not enabled by default.